### PR TITLE
[11.0][ADD]purchase_reception_notify

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This module will send a notification to the followers of the purchase order when a reception to satisfy that purchase order is performed. A message like the following will be generated:

image

This is inspired on purchase_request's notifications.

### CODE REVIEW

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1: squash commits

File: purchase_reception_notify/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                 purchase_line_id['stock_move'].product_qty,
+                 purchase_line_id['stock_move'].product_uom.name,
+                 )
+        message += '</ul>'
+        return message
+
+    @api.multi
+    def action_done(self):
+        super(StockPicking, self).action_done()
+        for picking in self:
```
Comment (MiquelRForgeFlow) [inline purchase_reception_notify/models/stock_picking.py]: `for picking in self.filtered(lambda p: picking.picking_type_id.code == 'incoming')`

File: purchase_reception_notify/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            purchase_dict = {}
+            if picking.picking_type_id.code != 'incoming':
+                continue
```
Comment (MiquelRForgeFlow) [inline purchase_reception_notify/models/stock_picking.py]: remove this two lines

File: purchase_reception_notify/models/stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            for move in picking.move_lines:
+                if move.purchase_line_id:
```
Comment (MiquelRForgeFlow) [inline purchase_reception_notify/models/stock_picking.py]:
```suggestion
            for move in picking.move_lines.filtered('purchase_line_id'):
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM code review

**Comment by HviorForgeFlow** [comment]
travis :red_circle:

File: purchase_reception_notify/models/stock_picking.py [hunk 4]
```diff
@@ -32,26 +32,24 @@ def _purchase_order_picking_confirm_message_content(
     @api.multi
     def action_done(self):
         super(StockPicking, self).action_done()
-        for picking in self:
+        for picking in self.filtered(
+                lambda p: picking.picking_type_id.code == 'incoming'):
```
Comment (MiquelRForgeFlow) [inline purchase_reception_notify/models/stock_picking.py]:
```suggestion
                lambda p: p.picking_type_id.code == 'incoming'):
```

**Pull request closed without merging**